lib/qx/tool/cli/commands/contrib/Install.js

```diff
@@ -68,164 +72,236 @@ qx.Class.define("qx.tool.cli.commands.contrib.Install", {
+              );
+            }
+            break;
+          default:
+            let lib = this.getCache().repos.data[lib_name];
+            if (!lib) {
+              throw new qx.tool.cli.Utils.UserError(`${lib_name} is not an contrib!`);
+            }
+            let versionList = [];
+            lib.releases.list.forEach(elem => versionList.push(elem.substring(1)));
+            let version = semver.maxSatisfying(versionList, lib_version, {loose: true});
+            if (!version) {
+              throw new qx.tool.cli.Utils.UserError(`No satisfying version found for ${lib_name}@${lib_version}!`);
+            }
+            await this.__installRepo(lib_name, `v${version}`);
+            break;
+        }
+      };
+    },
+
     installApplication: async function(repoName) {
       let contribJson = await this.getContribData();
       let libraryJson = contribJson.libraries.find(data => data.repo_name === repoName);
-      if (!libraryJson)
+      if (!libraryJson) {
         throw new Error("Repo for " + repoName + " is not installed as a contrib");
-      
-      let manifest = await qx.tool.compiler.utils.Json.loadJsonAsync(path.join(libraryJson.path, "Manifest.json"), "utf8");
+      }
+
+      let manifest = await qx.tool.compiler.utils.Json.loadJsonAsync(path.join(libraryJson.path, "Manifest.json"));
       if (!manifest.provides || !manifest.provides.application) {
-        if (this.argv.verbose) 
+        if (this.argv.verbose) {
           console.info(">>> No application to install.");
+        }
         return;
       }
-      
+
       let compileJson = await qx.tool.compiler.utils.Json.loadJsonAsync("compile.json");
       let manifestApp = manifest.provides.application;
       let app = compileJson.applications.find(app => {
-        if (manifestApp.name && app.name)
+        if (manifestApp.name && app.name) {
           return manifestApp.name === app.name;
+        }
         return manifestApp["class"] === app["class"];
       });
       if (!app) {
         compileJson.applications.push(manifestApp);
         app = manifestApp;
       } else {
-        for (let key in manifestApp)
+        for (let key in manifestApp) {
           app[key] = manifestApp[key];
+        }
       }
-      if (this.argv.verbose) 
+      if (this.argv.verbose) {
         console.info(">>> Installed application " + (app.name||app["class"]));
-      
+      }
+
       return await qx.tool.compiler.utils.Json.saveJsonAsync("compile.json", compileJson);
     },
 
     __loadFromContribJson: async function() {
-      // read libraries array from contrib.json 
+      // read libraries array from contrib.json
       let contrib_json_path = process.cwd() + "/contrib.json";
-      let data = fs.existsSync(contrib_json_path) ? jsonlint.parse( fs.readFileSync(contrib_json_path,"utf-8") ) : { libraries : [ ] };
+      let data = fs.existsSync(contrib_json_path) ? JSON.parse(fs.readFileSync(contrib_json_path, "utf-8")) : {libraries : []};
```

Comment:
This could be `qx.tool.compiler.utils.Json.parseJson` because `JSON.parse` does not support comments - although this is debatable given than `contrib.json` is typically edited by `qx` only, and when written out any comments would be lost.  

It *is* important for other `.json` files though (see review below)